src/arrayMethodSort.js

```diff
@@ -1,12 +1,24 @@
-'use strict';
+"use strict";
 
 /**
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b)
```

Comment:
compareFunction must return number not boolean, and better to create separate defaultCompareFunction